target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-rb3011uias.dts

```diff
@@ -0,0 +1,599 @@
+#include "qcom-ipq8064-v2.0.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "MikroTik RouterBOARD RB3011UiAS";
+	compatible = "mikrotik,rb3011uias", "qcom,ipq8064";
```

Comment:
This should probably be
	model = "MikroTik RouterBOARD 3011UiAS";
	compatible = "mikrotik,routerboard-3011uias", "qcom,ipq8064";